openapi.yaml

```diff
@@ -1446,18 +1464,24 @@ paths:
         | ------------|:----------------------------:|
         | MWA         | The Market weighted averages |
         | GWA         | The Global weighted averages |
+        | LX          | Liquid Indices               |
 
         Each index type will have an index id which identifies the constituent type used to build the index -
 
         | Index Type | Index Id  | Output |
         | -----------|:---------:|:---------:|
         | MWA        | Market Id | OHLCV for that market |
         | GWA        | Asset Id  | OHLCV for that coin / asset |
+        | LX         | Index Id  | OHLCV for Bitcoin Liquid Index (BLX)
+        | LX         | Index Id  | OHLCV for Ethereum Liquid Index (ELX)
+        | LX         | Index Id  | OHLCV for Ripple Liquid Index (XRPLX)
```

Comment:
Missing end column for LX, maybe use a standard template?

Reply:
Done